README.md

```diff
@@ -1,134 +1,142 @@
 [![Build Status](https://dev.azure.com/responsibleai/fairlearn/_apis/build/status/Nightly?branchName=master)](https://dev.azure.com/responsibleai/fairlearn/_build/latest?definitionId=23&branchName=master) ![MIT license](https://img.shields.io/badge/License-MIT-blue.svg) ![PyPI](https://img.shields.io/pypi/v/fairlearn?color=blue)
 
+# fairlearn
 
+The project `fairlearn` seeks to enable anyone involved in the development of artificial intelligence (AI) systems to assess their system's fairness and mitigate the observed unfairness. The fairlearn repository contains a Python package and Jupyter notebooks with the examples of usage.
 
-# Fairness in machine learning - Mitigation algorithms
+- [Current release](#current-release)
+- [What we mean by _fairness_](#what-we-mean-by-fairness)
+- [Overview of fairlearn](#overview-of-fairlearn)
+- [Install fairlearn](#install-fairlearn)
+- [Usage](#usage)
+- [Contributing](#contributing)
+- [Maintainers](#maintainers)
+- [Issues](#issues)
 
-A Python package that implements a variety of algorithms that mitigate unfairness in supervised machine learning.
+## Current release
 
-## Current Release
+- The current stable release is available at [fairlearn v0.3.0](https://github.com/fairlearn/fairlearn/tree/v0.3.0).
 
-* The current stable release is available at [FairLearn v0.2.0](https://github.com/fairlearn/fairlearn/tree/v0.2.0).
+- Our current version differs substantively from version 0.2 or earlier. If you were using these older versions, please visit our [onboarding guide](#onboarding-guide).
 
-* Our latest work differs substantively from version 0.2.0, please visit the repo for further information. In particular, look at [Existing users: How to onboard to fairlearn v0.3+](#existing).
+## What we mean by _fairness_
 
-#  Algorithms
+AI systems can behave unfairly for a variety of reasons. In fairlearn, we define whether an AI system is behaving unfairly in terms of its impact on people – i.e., in terms of harms. We focus on two kinds of harms:
```

Comment:
Should we put all of these `fairlearn`s in backquotes?

Reply:
nope. see my response above (frequent font changes are a bit disruptive).